<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MicroQQ(8888)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/favicon.ico</normaloff>:/new/prefix1/images/favicon.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_Bold">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>360</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/B_transparent.ico</normaloff>:/new/prefix1/images/B_transparent.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Italic">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/I.png</normaloff>:/new/prefix1/images/I.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Underline">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/U.png</normaloff>:/new/prefix1/images/U.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Color">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>颜色</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/color.png</normaloff>:/new/prefix1/images/color.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>531</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>531</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Fontsize">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>87</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_emit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>630</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Ctrl+E</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ESC</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>我的端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IP">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_myPort">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>对方端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_otherPort">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
